configs/task.yaml

```diff
@@ -4,4 +4,6 @@ create_plots: 1
 generate_samples: 1
 n_generate_samples: 10
 n_plot_samples: 5
-save_model: 1
\ No newline at end of file
+save_model: 1
+train: 1
+test: 0
```

Comment:
@abhishekabhishek , this `task.yaml` file is a good place for the `train` and `test` switches.